modules/http/http-parser.c

```diff
@@ -64,6 +65,8 @@ static CfgLexerKeyword http_keywords[] =
   { "body_prefix",      KW_BODY_PREFIX },
   { "body_suffix",      KW_BODY_SUFFIX },
   { "delimiter",        KW_DELIMITER },
+  { "accept_encoding",  KW_ACCEPT_ENCODING },
+  { "content_compression",    KW_MESSAGE_COMPRESSION },
```

Comment:
Shouldn't this be `"message_compression"`?

Reply:
I named it content compression because it might be less misunderstandable to the end user this way. To make the code more uniform, I would suggest renaming the keyword and associated variables instead to reflect "content compression".

Reply:
Done

Reply:
Thank you. Would it be possible also to rename the setter and the field, too? `http_dd_set_message_compression()`, `gint8 message_compression` Thanks!

Reply:
I found it more logical to use the "inner name" `message` to the programmer, as in this context it makes more sense to me, but to name it `content` for the end user, as it is less ambiguous. Is such practice allowed, or is it discouraged?

Reply:
I think it is preference.

It is allowed, but you would not see many examples of it, because we usually get confused, when we are debugging a `message-compression()` option, and we do not see a `message_compression` field in the driver. (Talking about personal experience, but I might be the only one with this problem :) )

Feel free to ignore this review comment. You can leave this as is.

Reply:
Valid concern. I will think about it.